api/graphql/schema.graphql

```diff
@@ -573,6 +573,8 @@ type Pool implements HasRoleAssignments {
       relation: "activities"
       scopes: ["whereIsPoolActivity", "authorizedToViewPoolActivity"]
     )
+  applicantsCount: Int
```

Comment:
I wanted to make it `Int!`
But that makes a massive codegen typing headache cuz of any usage of `: Pool` 😢 

<img width="1074" height="541" alt="image" src="https://github.com/user-attachments/assets/92d1edc7-41d6-4825-b010-66c6444142fa" />